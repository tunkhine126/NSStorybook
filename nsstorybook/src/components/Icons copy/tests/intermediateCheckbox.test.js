/**
 *
 * Tests for IntermediateCheckbox
 *
 * @see https://github.com/react-boilerplate/react-boilerplate/tree/master/docs/testing
 *
 */

import React from 'react';
import TestingComponent from 'components/shared/Testing';
import { render } from '@testing-library/react';
// import 'jest-dom/extend-expect'; // add some helpful assertions
import IntermediateCheckbox from '../intermediateCheckbox';

describe('<IntermediateCheckbox />', () => {
  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    render(
      <TestingComponent>
        <IntermediateCheckbox />
      </TestingComponent>
    );
    expect(spy).not.toHaveBeenCalled();
  });

  it('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <TestingComponent>
        <IntermediateCheckbox />
      </TestingComponent>
    );
    expect(firstChild).toMatchSnapshot();
  });
});
