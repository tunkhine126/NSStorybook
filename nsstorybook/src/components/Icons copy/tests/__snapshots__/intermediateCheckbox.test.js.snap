// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IntermediateCheckbox /> Should render and match the snapshot 1`] = `
<svg
  fill="none"
  height="16"
  viewBox="0 0 16 16"
  width="16"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    clip-rule="evenodd"
    d="M1.77778 0C0.8 0 0 0.8 0 1.77778V14.2222C0 15.2 0.8 16 1.77778 16H14.2222C15.2 16 16 15.2 16 14.2222V1.77778C16 0.8 15.2 0 14.2222 0H1.77778ZM3.28209 7.0324L12.7179 7.0324V8.77779H3.28209L3.28209 7.0324Z"
    fill="#00BAE0"
    fill-rule="evenodd"
  />
</svg>
`;
